### PR TITLE
Itinerary Endpoints done and 2 vacation endpoints done

### PR DESCRIPTION
**Pull request by lesleytomosada**

### CODE REVIEW

File: backend/index.ts [hunk 1]
```diff
@@ -13,9 +14,18 @@ app.use(
...
+
+
+app.get("/api/vacation/itinerary/:id", itineraryQueries.getItinerary);
+app.post("/api/vacation/itinerary/", itineraryQueries.createItineraryItem);
+app.put("/api/vacation/itinerary/item/:id", itineraryQueries.updateItineraryItem);
+app.delete("/api/vacation/itinerary/item/:id", itineraryQueries.deleteItineraryItem);
 
 app.listen(port, () => {
   console.log(`App running on port http://localhost:${port}`);
   console.log(itineraryQueries);
```
Comment (aidanliddiard) [inline backend/index.ts:29]: maybe delete these at some point :)

File: backend/src/itineraryQueries.js [hunk 2]
```diff
@@ -1,16 +1,109 @@
...
-    getTime
-  }
\ No newline at end of file
+    }
+  );
+};
+
+const updateItineraryItem = (request, response) => {
+  const id = parseInt(request.params.id);
+  console.log(request.body);
```
Comment (aidanliddiard) [inline backend/src/itineraryQueries.js:55]: another one :)

File: backend/src/vacationQueries.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
+const pool = require('./server.js');
+
+const getVacations = (request, response) => {
+    console.log(pool)
```
Comment (aidanliddiard) [inline backend/src/vacationQueries.js:4]: another one

**Review by tbyrd2237 — APPROVED** [review]

**Review by aidanliddiard — COMMENTED** [review]
just a few console.logs

**Review by aidanliddiard — APPROVED** [review]

File: backend/index.ts (hunk 1 above)
Comment (lesleytomosada) [reply backend/index.ts:29]: will do later on thanks!

**Pull request merged**